package.xml

```diff
@@ -13,6 +13,7 @@
 
   <buildtool_export_depend>cmake</buildtool_export_depend>
 
+  <depend>libpoco-dev</depend>
   <depend>pcre</depend>
   <depend>zlib</depend>
```

Comment:
pcre and zlib are pulled in by the upstream poco package so these are only needed when building poco with this package. I think in order to build it you'll also need libexpat1-dev and libsqlite3-dev. I can either add all build dependencies or remove them all.